boards/arm/stm32f411e_disco/doc/stm32f411e_disco.rst

```diff
@@ -0,0 +1,184 @@
+.. _stm32f411e_disco_board:
+
+STM32F411E-DISCO
+################
+
+Overview
+********
+
+The STM32F411E Discovery kit features an ARM Cortex-M4 based STM32F411VE MCU with a wide range of connectivity support and configurations Here are some highlights of the STM32F411E-DISCO board:
+
+
+- STM32F411VET6 microcontroller featuring 512 KB of Flash memory, 128 KB of RAM in an LQFP100 package
+- On-board ST-LINK/V2 with selection mode switch to use the kit as a standalone STLINK/V2 (with SWD connector for programming and debugging)
+- Board power supply: through USB bus or from an external 5 V supply voltage
+- External application power supply: 3 V and 5 V
+- L3GD20, ST MEMS motion sensor, 3-axis digital output gyroscope.
+- LSM303DLHC, ST MEMS system-in-package featuring a 3D digital linear acceleration sensor and a 3D digital magnetic sensor.
+- MP45DT02, ST MEMS audio sensor, omnidirectional digital microphone
+- CS43L22, audio DAC with integrated class D speaker driver
+- Eight LEDs:
+
+       - LD1 (red/green) for USB communication
```

Comment:
This looks better if you align the ``-`` for this indented list with the first character of the parent list:

     - Eight LEDs:

       - LD1 (red/green) ...

Reply:
I've reduced de indent a bit, but if align like you say, the HTML rendered by the markup does not show the list.
Thank you!